packages/next-starter/src/pages/_app.tsx

```diff
@@ -1,6 +1,5 @@
-import { makeServer } from '@onr/shared/mirage';
-
 if (process.env.NODE_ENV === 'development' && process.env.NEXT_PUBLIC_ENABLE_MIRAGE === 'true') {
+  const { makeServer } = require('@onr/shared/mirage');
   makeServer({ environment: 'development' });
 }
```

Comment:
I think we can keep the import syntax? although that would make the code looks a bit complex
```
(async() => {
   const {makeServer} = await import('@onr/shared/mirage');
   ....
})()
```

Reply:
or use Promise.then syntax